tests/cnf/ran/talm/tests/talm-batching.go

```diff
@@ -199,46 +199,47 @@ var _ = Describe("TALM Batching Tests", Label(tsparams.LabelBatchingTestCases),
 				Spoke2APIClient, tsparams.CatalogSourceName, tsparams.TemporaryNamespace).Exists()
 			Expect(catSrcExistsOnSpoke2).To(BeFalse(), "Catalog source exists on spoke 2")
 		})
+		It("should continue the CGU when the first batch fails with the Continue batch timeout action",
```

Comment:
@klaskosk is there a  way to format this without additional indentation in the test body? I found it's a bit unfortunate that if our test description exceeds certain number of chars, then the whole test body will be affected.

Reply:
@yliu127 I don't think there's any way to break it into multiple lines without then indent being formatted in.  I do try to edit down the length of the messages if it's just a few characters off

Reply:
Or maybe I'm wrong lol